src/screens/transaction-details/helpers/getFormattedEthFee.ts

```diff
@@ -0,0 +1,24 @@
+import { ethUnits } from '@/references';
+
+const _0_0001ETH_IN_WEI = 100000000000000;
+const _0_0001GWEI_IN_WEI = 100000;
+
+export function getFormattedEthFee(weiFee: number): string {
```

Comment:
this is super hard to follow, im also wondering if we dont already have a helper for this?